ta/arch/arm/link.mk

```diff
@@ -49,6 +53,7 @@ $(link-out-dir$(sm))/$(user-ta-uuid).elf: $(objs) $(libdeps) \
 					  $(link-script-pp$(sm))
 	@$(cmd-echo-silent) '  LD      $$@'
 	$(q)$(LD$(sm)) $(ldargs-$(user-ta-uuid).elf) -o $$@
+	$(q)$(if $(CFG_TA_FTRACE_SUPPORT), rm -f dyn_list)
```

Comment:
With a proper target as suggested above, this won't be necessary, you will add the file to `cleanfiles` instead.

Reply:
ok